pkg/services/ngalert/api/lotex_am.go

```diff
@@ -33,6 +34,20 @@ func NewLotexAM(proxy *AlertingProxy, log log.Logger) *LotexAM {
 	}
 }
 
+func (am *LotexAM) RouteGetAMStatus(ctx *models.ReqContext) response.Response {
+	return am.withReq(
+		ctx,
+		http.MethodGet,
+		withPath(
+			*ctx.Req.URL,
+			amStatusPath,
+		),
+		nil,
+		jsonExtractor(&apimodels.GettableStatus{}),
```

Comment:
I get a failure when trying to [encode and validate](https://github.com/grafana/grafana/blob/d76f08f3668502ca95cded87c8bc60e621447df8/pkg/services/ngalert/api/tooling/definitions/alertmanager.go#L423) `PostableApiAlertingConfig` because the route [here](https://github.com/grafana/grafana/blob/d76f08f3668502ca95cded87c8bc60e621447df8/pkg/services/ngalert/api/tooling/definitions/alertmanager.go#L471) is empty.

Reply:
Oh wow, this is a good shout - we need to skip that validation when the backend is Lotex because there is a chance that config might be nil.